javascript/javascript3/week1/giphy-api.js

```diff
@@ -0,0 +1,25 @@
+const gifKey = "t2bqf6dFtQjkVxvGZj9ldvsQ2Q0k1Gjp";
+const divContainer = document.getElementById("container");
+
+async function getGifs(word, howMany) {
+  // Visual Studio Code helped me to convert my function to async function, it works, so I decided to leave it like that
+  const response = await fetch(
+    `https://api.giphy.com/v1/gifs/search?q=${word}}&limit=${howMany}&api_key=${gifKey}`
```

Comment:
Use a variable for your base URL to make this shorter. Alternatively, make a function that formulates the query for you.